# Results

[Link to supplemental data](https://docs.google.com/spreadsheets/d/1l3QPyZNJN0Ed3uDyDlk2Ulwe7vfN\_DcNGFpiGytuOg4/edit?usp=sharing).

**The broad lunar soil stress response model**: Reveals circadian rhythm, starvation and hormonal regulation of defence metabolism.&#x20;

Link to [Source figure/narrative slide show](https://docs.google.com/presentation/d/1BFprcp7zzBdi1luFcqVtUXI25qJImtgwWsIGVaGkdPA/edit?usp=sharing) that needs description and code to be linked.

{% embed url="https://docs.google.com/presentation/d/1BFprcp7zzBdi1luFcqVtUXI25qJImtgwWsIGVaGkdPA/edit" %}

## Analysis of Stress Response in Plants Exposed to Lunar Soil Conditions

### Introduction

Recent empirical evidence suggests that lunar soil exerts considerable stress on plant systems, significantly impacting their growth rates and defense mechanisms. This document aims to elucidate the primary stress response clusters identified in plants subjected to lunar soil, focusing on transcriptional activity, signaling pathways, metabolic processes, and circadian rhythm alterations.

### Stress Response Clusters

#### 1. Transcriptional Activity

Key transcription factors, including WRYK33, DREB2, AP2, NAC6, Kan1, and GBox – all known to be responsive to stress conditions – are observed to be upregulated. This upregulation indicates an enhanced expression of genes involved in stress adaptation mechanisms.

#### 2. Signaling Pathways

* **Cold Signaling**: There is a notable downregulation in cold signaling, suggesting a reduced response to low temperatures.
* **Auxin and Cytokinin Biosynthesis**: Both of these growth-regulating hormones show increased biosynthesis, likely as a compensatory mechanism to mitigate stress effects.
* **Defense Signaling**: A pronounced increase in defense signaling pathways, including those against fungal pathogens, has been observed. This is coupled with an upregulation in response to acid stress and Jasmonic Acid (JA) signaling, highlighting a multi-faceted defense response.
* **Photosynthesis and Carbon Metabolism**: Both processes are downregulated, illustrating the diversion of energy from growth to defense mechanisms.

#### 3. Metabolic Processes

* **Defense Metabolism**: There is a significant increase in defense metabolism, indicating an active production of defense compounds.
* **Oxidative Phosphorylation (Oxphos)** and **ERAD Protein Recycling Pathway**: Both pathways are upregulated, illustrating an increased energy production and protein recycling mechanism, respectively.
* **Protein Translation**: A downregulation in protein translation and ribosomal components is observed, further signifying the plant's shift from growth to stress response.

#### 4. Circadian Rhythm Alteration

The circadian rhythm is notably altered, partially through the modulation of CCA1 and TOC1, which could affect various physiological processes in plants, including growth and stress responses.

### Summary

Exposure to lunar soil conditions imposes significant stress on plant systems, manifesting in slowed growth and a pronounced activation of secondary defense mechanisms. Key response mechanisms include the enhancement of flavonoid, anthocyanin metabolism, and iron transport; suppression of transcription; and a robust defense metabolism focusing on JA, Reactive Oxygen Species (ROS), hypoxia, and defense against biotic stressors. Additionally, alterations in circadian rhythm, cold signaling, hormone synthesis, and microtubule function underscore the broad impact of lunar soil on plant physiology.

In conclusion, understanding these stress response clusters provides critical insights into plant resilience mechanisms under extraterrestrial conditions, contributing to our knowledge of plant adaptation and survival strategies in harsh environments.



[DRB Summary slide with the Mini Groot Broad narrative](https://docs.google.com/presentation/u/0/d/1a\_\_b4exPY8by33kcYLlLfEo\_rVxcj7CM7w\_\_xbTRThE/edit)

{% embed url="https://docs.google.com/presentation/d/1a__b4exPY8by33kcYLlLfEo_rVxcj7CM7w__xbTRThE/edit" %}

**Note: These slides can be refined and exported from the slide show in a high resolution…**

![](../.gitbook/assets/7.png)

**Figure X**: Recent reanalyses of lunar samples have demonstrated varying degrees of stress-induced gene expression that strongly correlate with the geographic location of these samples on the Moon. This finding suggests a significant environmental influence on the molecular biology of organisms exposed to lunar conditions, warranting further investigation into the nature and implications of these variations.

**Note:** 50 pages below of other summary figures from the Broad lunar model.

![Higher resolution avalibale on slides in repo.](../.gitbook/assets/8.png)

**Figure x:** The signal indicative of the general adaptation response to starvation stress was pinpointed through GAGE (Generally Applicable Gene-set Enrichment) analysis focused on the KEGG (Kyoto Encyclopedia of Genes and Genomes) pathways associated with the ribosome and the phenylpropanoid biosynthesis pathway.



![Higher resolution avalibale on slides in repo.](../.gitbook/assets/9.png)

Figure XXX: Compare the significant DEG counts for various Apollo missions/sites based on phenotypic stress levels using a bar plot. Utilize the KEGG Pathview to illustrate these data on the KEGG plant hormonal pathways summary, highlighting consistent changes in hormonal regulation across all regolith types.

Each of the three locations exhibited a range of transcriptional responses, with APPolo 11 showing the greatest change in gene expression. Despite variations in the number of differentially expressed genes, the responsive pathways are broadly similar. These samples can be pooled to create a comprehensive lunar model comparing all lunar regolith samples to the simulant. We then used KEGG Pathview to illustrate changes in hormonal signaling pathways, revealing significant alterations in ethylene, gibberellin (GA), cytokinin, jasmonic acid (JA), salicylic acid (SA), and auxin. These stress signals interact with responses to blue light perception and circadian rhythm regulation in response to starvation.

![Tsne clusering identifies 4 clusers with disinct stress response functions.](../.gitbook/assets/16.png)

**4 clusters defined by Kmean**

#### Understanding the Aggregation of Lunar Samples: The Broad Lunar Model

In the pursuit of advancing our comprehension of the Moon's geology and its relationship with Earth, researchers have categorized lunar samples into a unified framework, referred to as the "Broad Lunar Model". This approach synthesizes the diverse array of samples gathered from the lunar surface into a cohesive linear model, facilitating a more comprehensive evaluation of the Moon's composition and its evolutionary linkage with Earth.

![Figure x: An alternative statical model that groups all lunar locations can be used to identify DEG’s. Box plot, densit plot, scatter plot and MA plot all show transcriptional data distrubtion. Tsne and MDS plots show grouping that doesn’t correlate with main factors (location).](../.gitbook/assets/17.png)



![Figure X: DEG, Up count, DEG down count, volcano plot summary of FC and adjusted p-values and Genome ontology terms associated with the 5 main principle components within the expression matrix. All lunar samples were binned together to create the broad lunar response model.](../.gitbook/assets/18.png)



#### Comparative Analysis of True Lunar Regolith and JSC Simulate: Gene Expression Insights

***

This study presents a comprehensive analysis comparing the gene expression profiles of organisms exposed to true lunar regolith and those subjected to the JSC-1A lunar simulant. Our analysis identified 759 loci with suppressed expression and 631 loci with enhanced expression in the presence of true lunar regolith, suggesting significant alterations in various biological processes.

**Key Findings**

* **Suppression of Gene Expression**: A notable suppression of gene expression was observed across 759 loci, indicating a potential downregulation of key biological functions in response to true lunar regolith exposure.
* **Enhanced Gene Expression**: Conversely, 631 loci exhibited enhanced expression, suggesting adaptive or stress response mechanisms might be activated by the lunar environment.
* **Biological Processes Affected**: The analysis reveals substantial impacts on:
  * Ribosome composition
  * mRNA processing
  * DNA repair mechanisms
  * Cell cycle regulation
  * Defence mechanisms
  * Photosynthesis processes

**Gene Ontology (GO) Analysis**

A detailed Gene Ontology (GO) analysis provided further insights into the affected biological processes, highlighting significant changes with associated p-values indicating statistical relevance. The GO analysis underscores the extensive impact of lunar regolith exposure on cellular and molecular components, elucidating potential pathways affected by the lunar environment. Gene ontology is a project that produces a dynamic annotated database that defines biological processes, cellular components, and molecular functions of gene products in a species-independent manner \[1]. This particular graph is focus on biological processes.

![Dendrogram showing connected GO groups identified by the broad lunar response model. Biological process differentially expressed Earth JSC simulant vs Lunar regolith](../.gitbook/assets/19.png)

In the graph, terms are linked by dendrograms indicating parent-child relationships. A child term inherits its parent's meaning and adds specificity. For instance, "mitotic cell cycle phase transition" (6e-16) is a child of "cell cycle phase transition" (4e-16), showing that the former is a specific type of the latter. The numbers next to the terms likely represent p-values, indicating the probability that the observed association between genes and a term occurred by chance. Lower p-values suggest a stronger association.

The visualization, derived from the detailed Gene Ontology (GO) term graph by the GO consortium\[1], categorizes biological processes from broad to specific. It begins with the Cell Cycle, differentiating between the Meiotic Cell Cycle, including the Meiotic Cell Cycle Process, and the Mitotic Cell Cycle, which subdivides into the Mitotic Cell Cycle Process and the Mitotic Cell Cycle Phase Transition. Another category, the Response to External Stimulus, includes the Response to Chemical, encompassing the Response to Hormone, and the Response to Other Organism, including the Defense Response and the Response to Fungus.



### Identification of 4 Gene Clusters Through WGCNA

<figure><img src="../.gitbook/assets/20.png" alt=""><figcaption><p>Ideniftifcation of Gene Clusters through Weighted Gene Co-expression Network Analysis,</p></figcaption></figure>

Using Weighted Gene Co-expression Network Analysis (WGCNA), four distinct clusters have been identified based on their functional grouping. These clusters are delineated within the dendrogram that represents both gene expression and functional similarities. The clusters are categorized as follows: Turquoise, Brown, Yellow, and Blue. Each of these clusters has been characterized by the top 10 loci that define their respective grouping, providing key insights into the underlying gene expressions that demarcate each cluster.



**A few example "marker" loci from the 4 clusters identified by WGCNA**

**Turquoise:** GLR1.3, PER34 CEN2 and NAC047.

**Brown:** NRT1.1, PRX11C, WRK60, and TRX9

**Yellow:** SUS3, RABC2B, TSA1, BGLU18, GOLS1, PHO1-H10 and GLO5

**Blue:** SOD2, PAP24, UP6, and FAP2



### **Conclusion**

This comparative study between true lunar regolith and JSC-1A lunar simulant underscores the profound effects of the lunar material on gene expression. Not only does this research enhance our understanding of life's resilience in extraterrestrial environments, but it also provides critical insights for future lunar explorations and the development of bio-regenerative life support systems.

**Future Directions**

Further research is necessary to validate these findings and explore their implications for long-term lunar habitation and the viability of cultivating life-supporting ecosystems on the Moon.



**Table X: KEGG pathway enrichment**

| **Direction**  | **adj.Pval** | **nGenes** | **KEGG Pathways**                                                                |
| -------------- | ------------ | ---------- | -------------------------------------------------------------------------------- |
| Down regulated | 3.6e-03      | 9          | [Homologous recombination](http://www.genome.jp/kegg-bin/show\_pathway?ath03440) |
|                | 3.6e-03      | 7          | [Circadian rhythm](http://www.genome.jp/kegg-bin/show\_pathway?ath04712)         |

**Table X: AraCyc pathway enrichment**

| **Direction**  | **adj.Pval** | **nGenes** | **Ara Cyc Pathways**                   |
| -------------- | ------------ | ---------- | -------------------------------------- |
| Down regulated | 2.6e-05      | 8          | Thioredoxin pathway                    |
| Up regulated   | 3.7e-03      | 5          | Indole glucosinolate activation        |
|                | 3.7e-03      | 8          | Glutathione-mediated detoxification II |

**Table X: Reactome pathway enrichment**

| **Direction** | **adj.Pval** | **nGenes** | **Reactome Pathways**                                                                                            |
| ------------- | ------------ | ---------- | ---------------------------------------------------------------------------------------------------------------- |
| Up regulated  | 2.7e-03      | 5          | [R-ATH-1119473.1 Cytokinins-O-glucoside biosynthesis](http://plantreactome.gramene.org/index.php?lang=en)        |
|               | 2.7e-03      | 4          | [R-ATH-8879007.1 Response to cold temperature](http://plantreactome.gramene.org/index.php?lang=en)               |
|               | 6.2e-03      | 2          | [R-ATH-1119580.1 IAA biosynthesis II](http://plantreactome.gramene.org/index.php?lang=en)                        |
|               | 6.2e-03      | 3          | [R-ATH-1119438.1 Secologanin and strictosidine biosynthesis](http://plantreactome.gramene.org/index.php?lang=en) |

Table X: Transcription factor enrichment database STIF

| **Direction** | **adj.Pval** | **nGenes** | **STIF TFDB Pathways**                                                                                |
| ------------- | ------------ | ---------- | ----------------------------------------------------------------------------------------------------- |
| Up regulated  | 4.0e-11      | 67         | [G box bHLH target](http://caps.ncbs.res.in/cgi-bin/mini/databases/stifdb2/browse\_by\_tfacts.pl)     |
|               | 1.2e-10      | 112        | [W box WRKY target](http://caps.ncbs.res.in/cgi-bin/mini/databases/stifdb2/browse\_by\_tfacts.pl)     |
|               | 8.5e-09      | 120        | [HSE1 HSF target](http://caps.ncbs.res.in/cgi-bin/mini/databases/stifdb2/browse\_by\_tfacts.pl)       |
|               | 8.5e-09      | 114        | [Myb box1 MYB target](http://caps.ncbs.res.in/cgi-bin/mini/databases/stifdb2/browse\_by\_tfacts.pl)   |
|               | 1.4e-08      | 73         | [Myb box3 MYB target](http://caps.ncbs.res.in/cgi-bin/mini/databases/stifdb2/browse\_by\_tfacts.pl)   |
|               | 3.9e-05      | 51         | [DREB AP2 EREBP target](http://caps.ncbs.res.in/cgi-bin/mini/databases/stifdb2/browse\_by\_tfacts.pl) |
|               | 4.4e-05      | 53         | [Nac box NAC target](http://caps.ncbs.res.in/cgi-bin/mini/databases/stifdb2/browse\_by\_tfacts.pl)    |
|               | 6.8e-05      | 17         | [HBE HB target](http://caps.ncbs.res.in/cgi-bin/mini/databases/stifdb2/browse\_by\_tfacts.pl)         |
|               | 2.0e-04      | 64         | [AuxRE ARF target](http://caps.ncbs.res.in/cgi-bin/mini/databases/stifdb2/browse\_by\_tfacts.pl)      |
|               | 4.5e-04      | 49         | [G box2 bZIP target](http://caps.ncbs.res.in/cgi-bin/mini/databases/stifdb2/browse\_by\_tfacts.pl)    |
|               | 2.1e-03      | 33         | [Myb box4 MYB target](http://caps.ncbs.res.in/cgi-bin/mini/databases/stifdb2/browse\_by\_tfacts.pl)   |
|               | 2.6e-03      | 33         | [N box bHLH target](http://caps.ncbs.res.in/cgi-bin/mini/databases/stifdb2/browse\_by\_tfacts.pl)     |
|               | 8.7e-03      | 41         | [Myb box2 MYB target](http://caps.ncbs.res.in/cgi-bin/mini/databases/stifdb2/browse\_by\_tfacts.pl)   |

Table X: Transcription factor enrichment database GTRD

| **Direction**  | **adj.Pval** | **nGenes** | **TF targets from GTRD**                                                             |
| -------------- | ------------ | ---------- | ------------------------------------------------------------------------------------ |
| Down regulated | 2.0e-76      | 199        | [MYB3R3-Transcription factor MYB3R-3 target gene](http://gtrd.biouml.org/)           |
|                | 1.6e-40      | 105        | [RBR1-Retinoblastoma-related protein 1 target gene](http://gtrd.biouml.org/)         |
|                | 1.4e-18      | 68         | [E2FA-Transcription factor E2FA target gene](http://gtrd.biouml.org/)                |
|                | 1.1e-11      | 216        | [CCA1-Protein CCA1 target gene](http://gtrd.biouml.org/)                             |
|                | 9.7e-07      | 112        | [LHY-Protein LHY target gene](http://gtrd.biouml.org/)                               |
|                | 1.3e-03      | 181        | [ARR14-Two-component response regulator ARR14 target gene](http://gtrd.biouml.org/)  |
| Up regulated   | 9.7e-12      | 202        | [WRKY33-Probable WRKY transcription factor 33 target gene](http://gtrd.biouml.org/)  |
|                | 2.3e-08      | 154        | [WRKY18-WRKY transcription factor 18 target gene](http://gtrd.biouml.org/)           |
|                | 2.4e-07      | 138        | [WRKY40-Probable WRKY transcription factor 40 target gene](http://gtrd.biouml.org/)  |
|                | 1.6e-03      | 59         | [ATHB-6-Homeobox-leucine zipper protein ATHB-6 target gene](http://gtrd.biouml.org/) |
|                | 3.3e-03      | 99         | [KAN1-Transcription repressor KAN1 target gene](http://gtrd.biouml.org/)             |

DRB factors model Version 10

CORE SUPPRESSED 24 LOCI

Table: Enriched papers about suppressed core response is the "cold" or "blue light" slow growth response.

| **Blue Light- and Low Temperature-Regulated COR27 and COR28 Play Roles in the Arabidopsis Circadian Clock.**                               | **0.00003702781658218090** | **AT4G33980,AT5G24470,AT5G42900** | **27837007** |
| ------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------- | --------------------------------- | ------------ |
| **A Bayesian approach for parameter estimation in the extended clock gene circuit of Arabidopsis thaliana.**                               | 0.0005174829475177940      | AT2G40080,AT3G46640,AT5G24470     | 24267177     |
| **Two new clock proteins, LWD1 and LWD2, regulate Arabidopsis photoperiodic flowering.**                                                   | 0.0020294138224070200      | AT1G68050,AT2G21660,AT2G40080     | 18676661     |
| **Functional redundancy and new roles for genes of the autonomous floral-promotion pathway.**                                              | 0.008911494884438870       | AT2G33830,AT5G42900,AT5G48250     | 18408043     |
| **Molecular characterization of the Brassica rapa auxin-repressed, superfamily genes, BrARP1 and BrDRM1.**                                 | 0.015323319353979200       | AT1G28330,AT2G33830               | 23065269     |
| **Actinidia DRM1--an intrinsically disordered protein whose mRNA expression is inversely correlated with spring budbreak in kiwifruit.**   | 0.015323319353979200       | AT1G28330,AT2G33830               | 23516402     |
| **DRM1 and DRM2 expression regulation: potential role of splice variants in response to stress and environmental factors in Arabidopsis.** | 0.015323319353979200       | AT1G28330,AT2G33830               | 24442277     |
| **COR27 and COR28 encode nighttime repressors integrating Arabidopsis circadian clock and cold response.**                                 | 0.015323319353979200       | AT4G33980,AT5G42900               | 27990760     |
| **COR27 and COR28 Are Novel Regulators of the COP1-HY5 Regulatory Hub and Photomorphogenesis in Arabidopsis.**                             | 0.015323319353979200       | AT4G33980,AT5G42900               | 32769132     |
| **COLD REGULATED 27 and 28 are targets of CONSTITUTIVELY PHOTOMORPHOGENIC 1 and negatively affect phytochrome B signalling.**              | 0.04594511456698950        | AT4G33980,AT5G42900               | 32890447     |

Enriched protein domain’s also indicate suppression of “cold response loci/pathway”

|   | CCT domain \[[IPR010402](http://www.ebi.ac.uk/interpro/IEntry?ac=IPR010402)]                   | 2.374895e-4 | 4 |
| - | ---------------------------------------------------------------------------------------------- | ----------- | - |
|   | Cold-regulated protein 27/28 \[[IPR044678](http://www.ebi.ac.uk/interpro/IEntry?ac=IPR044678)] | 0.006015    | 2 |



Figure x: Very detailed Metascape analysis of the location-specific distinction available on request.

###
